### ISSUE TITLE
Select item programmatically

### ISSUE THREAD

**Issue by hackeurtalibe** [toystars/react-native-multiple-select#18]
Hi, thank for this select plugin, i would like to select one item programmatically, is there any way to do that please

**Comment by toystars** [comment]
@hackeurtalibe yes there is a way. The `selectedItems` prop is a control prop. So if you update the array of ids being passed, it will update whatever is selected in the component.

**Comment by camux** [comment]
SelectedItems fail with infinite loop when we set attribute single={false}, I can't set default values.

**Comment by toystars** [comment]
@camux I've not been able to reproduce that. Maybe you can pull the latest release and see if the issue still persists
